### PR TITLE
fix: adjusting deploy with cloudbuild docs for examples

### PR DESCRIPTION
**Pull request by caetano-colin**

### CODE REVIEW

File: 5-appinfra/README.md [hunk 1]
```diff
@@ -27,6 +27,85 @@ You may add additional infrastructure like application-specifc databases or othe
 
 ## Usage
 
+### Deploying with Google Cloud Build
+
+The steps below assume that you are checkout out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```
Comment (apeabody) [inline 5-appinfra/README.md:32]: ?

```suggestion
The steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```

File: 5-appinfra/README.md [hunk 2]
```diff
@@ -27,6 +27,85 @@ You may add additional infrastructure like application-specifc databases or othe
...
+
+1. Use `terraform output` to get the state bucket value from 1-bootstrap output and replace the placeholder in `terraform.tfvars`.
+
+   ```bash
+   terraform -chdir="./terraform-google-enterprise-application/1-bootstrap/" init
+   export remote_state_bucket=$(terraform -chdir="./terraform-google-enterprise-application/1-bootstrap/" output -raw state_bucket)
+   echo "remote_state_bucket = ${remote_state_bucket}"
+   ```
+
+1. Clone the repositories for each service and initialized:
```
Comment (apeabody) [inline 5-appinfra/README.md:67]: ?

```suggestion
1. Clone the repositories for each service and initialize:
```

File: examples/cymbal-bank/README.md [hunk 3]
```diff
@@ -418,18 +431,26 @@ The steps below assume that you are checkout out on the same level as `terraform
 
 #### Application Source Code pipeline
 
-The step below assumes your are on `terraform-google-enterprise-application` repository.
+The steps below assume that you are checkout out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```
Comment (apeabody) [inline examples/cymbal-bank/README.md:434]: ?
```suggestion
The steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```

File: examples/cymbal-bank/README.md [hunk 4]
```diff
@@ -993,7 +1034,14 @@ The steps below assume that you are checkout out on the same level as `terraform
...
```
Comment (apeabody) [inline examples/cymbal-bank/README.md:1037]: ?
```suggestion
The steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```

File: examples/cymbal-shop/README.md [hunk 5]
```diff
@@ -127,6 +138,130 @@ The steps below assume that you are checkout out on the same level as `terraform
     cd ../
     ```
 
-#### Add Cymbal shop envs at App Infra
+#### Deploy Cymbal Shop App Infra
+
+This stage will create the CI/CD pipeline for the service, and application specific infrastructure if specified.
+
+**IMPORTANT**: The steps below assume that you are checkout out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```
Comment (apeabody) [inline examples/cymbal-shop/README.md:145]: ?
```suggestion
**IMPORTANT**: The steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```

File: examples/cymbal-shop/README.md [hunk 6]
```diff
@@ -127,6 +138,130 @@ The steps below assume that you are checkout out on the same level as `terraform
...
+
+```txt
+.
+├── terraform-example-foundation
+├── terraform-google-enterprise-application
+└── .
+```
+
+1. Retrieve Cymbal Shop repository, Admin Project and Appplication specific State Bucket that were created on 4-appfactory stage.
```
Comment (apeabody) [inline examples/cymbal-shop/README.md:154]:
```suggestion
1. Retrieve Cymbal Shop repository, Admin Project, and Application specific State Bucket that were created on 4-appfactory stage.
```

File: examples/cymbal-shop/README.md [hunk 7]
```diff
@@ -127,6 +138,130 @@ The steps below assume that you are checkout out on the same level as `terraform
...
+   ```bash
+    git checkout -b production
+    git push --set-upstream origin production
+    ```
+
+1. You can view the build results on Google Cloud Build at the admin project.
+
+#### Deploy Cymbal Shop App Source
+
+**IMPORTANT**: The steps below assume that you are checkout out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```
Comment (apeabody) [inline examples/cymbal-shop/README.md:225]: ?
```suggestion
**IMPORTANT**: The steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```

File: examples/multitenant-applications/README.md [hunk 8]
```diff
@@ -37,141 +43,713 @@ The steps below assume that you are checkout out on the same level as `terraform
...
+   ```bash
+    git checkout -b production
+    git push --set-upstream origin production
+    ```
+
+##### Cymbal Shop 5-appinfra
+
+This stage will create the CI/CD pipeline for the service, and application specific infrastructure if specified.
+
+**IMPORTANT**: The steps below assume that you are checkout out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```
Comment (apeabody) [inline examples/multitenant-applications/README.md:464]: ?
```suggestion
**IMPORTANT**: The steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` and `terraform-example-foundation` directories.
```

File: examples/multitenant-applications/README.md [hunk 9]
```diff
@@ -37,141 +43,713 @@ The steps below assume that you are checkout out on the same level as `terraform
...
+
+```txt
+.
+├── terraform-example-foundation
+├── terraform-google-enterprise-application
+└── .
+```
+
+1. Retrieve Cymbal Shop repository, Admin Project and Appplication specific State Bucket that were created on 4-appfactory stage.
```
Comment (apeabody) [inline examples/multitenant-applications/README.md:473]:
```suggestion
1. Retrieve Cymbal Shop repository, Admin Project, and Application specific State Bucket that were created on 4-appfactory stage.
```

**Review by apeabody — APPROVED** [review]
Thanks @caetano-colin!  A few notes below.

**Comment by caetano-colin** [comment]
@apeabody , thank you for the review! the typos should be fixed now

**Review by apeabody — APPROVED** [review]
Thanks @caetano-colin! LGTM

**Pull request merged**